src/domains/messages-common/03_database.tf

```diff
@@ -286,4 +286,30 @@ resource "azurerm_cosmosdb_sql_container" "remote_content_configuration" {
       path = "/\"_etag\"/?"
     }
   }
-}
\ No newline at end of file
+}
+
+resource "azurerm_cosmosdb_sql_container" "message-configuration" {
```

Comment:
Here it should be snake case.

Reply:
We are not using snake case for the name in other resources, not in this file at least

Reply:
[67dd2f7](https://github.com/pagopa/io-infra/pull/844/commits/67dd2f75ac3a6ae9a3e98a8e93b84f95b25cd4ef)